Makefile

```diff
@@ -51,10 +51,7 @@ setup_waffle_switches:
 # Run makemessages
 messages:
```

Comment:
I think it's potentially dangerous to conflate these 2 steps, since `makemessages` can generate message strings that haven't been reviewed by a human.

Reply:
> Then we can make sure the Dockerfile and github workflow only run compilemessages.

I have made this change, so it's all reproducible